tests/integration/test_events.py

```diff
@@ -1236,6 +1241,74 @@ def get_message(queue_url):
         # clean up
         self.cleanup(TEST_EVENT_BUS_NAME, rule_name, target_id, queue_url=queue_url)
 
+    @pytest.mark.parametrize(
+        "schedule_expression", ["rate(1 minute)", "rate(1 day)", "rate(1 hour)"]
+    )
+    @pytest.mark.aws_validated
+    def test_create_rule_with_one_unit_in_singular_should_succeed(
+        self, events_client, schedule_expression
+    ):
+        rule_name = self.create_rule_name()
+
+        # rule should be creatable with given expression
+        events_client.put_rule(Name=rule_name, ScheduleExpression=schedule_expression)
+
+        self.cleanup(rule_name=rule_name, events_client=events_client)
```

Comment:
Careful(!). The resources here won't be cleaned up when anything before raises an Exception. Generally these should be wrapped in try/finally (or use the fixture generator/lifecycle mechanism to clean up resources).